frappe/handler.py

```diff
@@ -165,11 +165,18 @@ def upload_file():
 	content = None
 
 	if 'file' in files:
+		ALLOWED_EXTENSION = ['jpeg', 'jpg', 'pdf', 'txt', 'png', 'gif', 'doc', 'docx', 'ppt', 'pptx', 'xls', 'xlsx',
```

Comment:
Should we Move this to the top?
with this
https://github.com/frappe/frappe/pull/16049/files#diff-1748e940618a9e8f25ac31dc72e029c509bcf12e9a1fb3aeeeaa59ffeecefacdR18

Reply:
Sure. Will do that.